### PR TITLE
Module 15 Examples finished, Image need to be finished

### PR DESCRIPTION
**Pull request by PINGXIANG**

### CODE REVIEW

**Comment by PINGXIANG** [comment]
Hahaha eventually.^-^

File: problemsets-mat223/modules/module15.tex [hunk 1]
```diff
@@ -111,7 +118,27 @@
 \begin{example}
 	Find the eigenvectors/values of $A=\mat{1&2\\3&2}$.
 
-	XXX Finish
+	Similarly to the above example, we first determinate $\Char(A)$:
+	\begin{align*}
+	    \Char(A)&=\det\left(\mat{1-\lambda&2\\3&2-\lambda}\right)\\
+	            &=(1-\lambda)(2-\lambda)-6=\lambda^2-3\lambda-4=(\lambda-4)(\lambda+1)
+	\end{align*}
+	Next, we set the $\chr(A)=0$ to find eigenvalues, which are $\lambda_1=-1$, $\lambda_2=4$. Based on the previous discussion, we know non-zero vectors in $\Null(A-\lambda_1I)$ are eigenvectors with eigenvalue $-1$. Computing,
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: `$\lambda_1=-1$, $\lambda_2=4$` -> `$\lambda_1=-1$ and $\lambda_2=4$`

File: problemsets-mat223/modules/module15.tex [hunk 2]
```diff
@@ -111,7 +118,27 @@
 \begin{example}
 	Find the eigenvectors/values of $A=\mat{1&2\\3&2}$.
 
-	XXX Finish
+	Similarly to the above example, we first determinate $\Char(A)$:
+	\begin{align*}
+	    \Char(A)&=\det\left(\mat{1-\lambda&2\\3&2-\lambda}\right)\\
+	            &=(1-\lambda)(2-\lambda)-6=\lambda^2-3\lambda-4=(\lambda-4)(\lambda+1)
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: Let's factor the other way `(4-\lambda)(-1-\lambda)`. This way we avoid having an extra `-` sign when doing 3x3 matrices.

File: problemsets-mat223/modules/module15.tex [hunk 3]
```diff
@@ -111,7 +118,27 @@
...
+	\end{align*}
+	Next, we set the $\chr(A)=0$ to find eigenvalues, which are $\lambda_1=-1$, $\lambda_2=4$. Based on the previous discussion, we know non-zero vectors in $\Null(A-\lambda_1I)$ are eigenvectors with eigenvalue $-1$. Computing, 
+	\[
+	\Null(A-\lambda_1I) = \Null\left(\mat{2&2\\3&3}\right)=
+	\Span\Set*{\mat{1\\-1}}.
+	\]
+	The vector $\mat{1\\-1}$ can be found by using row reduction to solve $\mat{2&2\\3&3}\vec x=\vec 0$.
+	
+	We conclude, one eigenvector of A with eigenvalue $\lambda_1=-1$ is $\mat{1\\-1}$. More generally, vectors $\vec v = k \mat{1\\-1}$ where $k \neq 0$ are eigenvectors of A with eigenvalue $-1$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: `A` -> `$A$`

File: problemsets-mat223/modules/module15.tex [hunk 4]
```diff
@@ -111,7 +118,27 @@
...
+	\Span\Set*{\mat{1\\-1}}.
+	\]
+	The vector $\mat{1\\-1}$ can be found by using row reduction to solve $\mat{2&2\\3&3}\vec x=\vec 0$.
+	
+	We conclude, one eigenvector of A with eigenvalue $\lambda_1=-1$ is $\mat{1\\-1}$. More generally, vectors $\vec v = k \mat{1\\-1}$ where $k \neq 0$ are eigenvectors of A with eigenvalue $-1$. 
+	
+	Similarly,
+	\[
+	\Null(A-\lambda_2I) = \Null\left(\mat{-3&2\\3&-2}\right)=
+	\Span\Set*{\mat{1\\-1}},
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: Not the right vector

File: problemsets-mat223/modules/module15.tex [hunk 5]
```diff
@@ -127,6 +154,27 @@
...
+	find $\Null(A-\lambda_1I)$ and $\Null(A-\lambda_2I)$.
+	
+	We will start by applying row reduction to $A-\lambda_1I$.
+	\begin{align*}
+	    \mat{1-\frac{5-\sqrt{33}}{2}&2\\3&4-\frac{5-\sqrt{33}}{2}} & \sim
+	    \mat{\frac{-3+\sqrt{33}}{2}&2\\3&\frac{3+\sqrt{33}}{2}} \sim \mat{1&\frac{4}{-3+\sqrt{33}}\\1&\frac{3+\sqrt{33}}{6}}\\
+	    & \sim \matc{1&\frac{4(3+\sqrt{33})}{(-3+\sqrt{33})(3+\sqrt{33})}\\1&\frac{3+\sqrt{33}}{6}} \sim \mat{1&\frac{3+\sqrt{33}}{6}\\1&\frac{3+\sqrt{33}}{6}}\\
+	    & \sim \matc{1&\frac{3+\sqrt{33}}{6}\\0&0}
+	\end{align*}
+	Thus, we conclude that one eigenvector corresponding to eigenvalue $\frac{5-\sqrt{33}}{2}$ is $\vec v_1 = \matc{\frac{3+\sqrt{33}}{6}\\-1}$. Similarly, one eigenvector corresponding to eigenvalue $\frac{5+\sqrt{33}}{2}$ is $\vec v_2=\matc{\frac{3-\sqrt{33}}{6}\\-1}$ is one eigenvector with eigenvalue $\lambda_2 = \frac{5+\sqrt{33}}{2}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: Let's change the structure of this sentence to something like:

```
Thus, we conclude that XXX is an eigenvector with eigenvalue YYY. Similarly, ...
```

File: problemsets-mat223/modules/module15.tex [hunk 6]
```diff
@@ -89,8 +88,16 @@
 
 \begin{example}
 	Find the characteristic polynomial of $A=\mat{1&2\\3&4}$.
-
-	XXX Finish
+	
+	By the definition of Characteristic polynomial of A, we have:
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: `Characteristic` -> `the characteristic`

Remove `:`

File: problemsets-mat223/modules/module15.tex [hunk 7]
```diff
@@ -89,8 +88,16 @@
...
 \begin{example}
 	Find the characteristic polynomial of $A=\mat{1&2\\3&4}$.
-
-	XXX Finish
+	
+	By the definition of Characteristic polynomial of A, we have: 
+	\begin{align*}
+	    \chr(A) &=\det(A-\lambda I)\\
+	            &=\det\left(\mat{1&2\\3&4}-\mat{\lambda&0\\0&\lambda}\right)=\det\left(\mat{1-\lambda&2\\3&4-\lambda}\right)\\
+	            &=(1-\lambda)(4-\lambda) - 6=\lambda^2 -5\lambda-2
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: Put a `.` at the end of this sentence.

File: problemsets-mat223/modules/module15.tex [hunk 8]
```diff
@@ -89,8 +88,16 @@
 
 \begin{example}
 	Find the characteristic polynomial of $A=\mat{1&2\\3&4}$.
-
-	XXX Finish
+	
+	By the definition of Characteristic polynomial of A, we have: 
+	\begin{align*}
+	    \chr(A) &=\det(A-\lambda I)\\
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: `\chr` -> `\Char`

File: problemsets-mat223/modules/module15.tex [hunk 9]
```diff
@@ -89,8 +88,16 @@
...
+	            &=\det\left(\mat{1&2\\3&4}-\mat{\lambda&0\\0&\lambda}\right)=\det\left(\mat{1-\lambda&2\\3&4-\lambda}\right)\\
+	            &=(1-\lambda)(4-\lambda) - 6=\lambda^2 -5\lambda-2
+	\end{align*}
+	Thus, we get the characteristic polynomial of A as $\Char(A)=\lambda^2 -5\lambda-2$.
+	
+
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: No need for these blank lines

File: problemsets-mat223/modules/module15.tex [hunk 10]
```diff
@@ -89,8 +88,16 @@
...
-
-	XXX Finish
+	
+	By the definition of Characteristic polynomial of A, we have: 
+	\begin{align*}
+	    \chr(A) &=\det(A-\lambda I)\\
+	            &=\det\left(\mat{1&2\\3&4}-\mat{\lambda&0\\0&\lambda}\right)=\det\left(\mat{1-\lambda&2\\3&4-\lambda}\right)\\
+	            &=(1-\lambda)(4-\lambda) - 6=\lambda^2 -5\lambda-2
+	\end{align*}
+	Thus, we get the characteristic polynomial of A as $\Char(A)=\lambda^2 -5\lambda-2$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: Delete this sentence. It is uneeded.

File: problemsets-mat223/modules/module15.tex (hunk 5 above)
Comment (PINGXIANG) [reply problemsets-mat223/modules/module15.tex]: Sure, I also feel that corresponding to is too long ^-^

File: problemsets-mat223/modules/module15.tex (hunk 8 above)
Comment (PINGXIANG) [reply problemsets-mat223/modules/module15.tex]: \Char will give a symbol like T(A) instead of char(A). Should I use \Char instead of \char?

Comment (PINGXIANG) [reply problemsets-mat223/modules/module15.tex]: Never mind... I have no idea what happened to my computer... ...

File: problemsets-mat223/modules/module15.tex [hunk 11]
```diff
@@ -175,6 +171,6 @@
 	    & \sim \matc{1&\frac{4(3+\sqrt{33})}{(-3+\sqrt{33})(3+\sqrt{33})}\\1&\frac{3+\sqrt{33}}{6}} \sim \mat{1&\frac{3+\sqrt{33}}{6}\\1&\frac{3+\sqrt{33}}{6}}\\
 	    & \sim \matc{1&\frac{3+\sqrt{33}}{6}\\0&0}
 	\end{align*}
-	Thus, we conclude that one eigenvector corresponding to eigenvalue $\frac{5-\sqrt{33}}{2}$ is $\vec v_1 = \matc{\frac{3+\sqrt{33}}{6}\\-1}$. Similarly, one eigenvector corresponding to eigenvalue $\frac{5+\sqrt{33}}{2}$ is $\vec v_2=\matc{\frac{3-\sqrt{33}}{6}\\-1}$ is one eigenvector with eigenvalue $\lambda_2 = \frac{5+\sqrt{33}}{2}$.
+	Thus, we conclude that one eigenvector with eigenvalue $\frac{5-\sqrt{33}}{2}$ is $\vec v_1 = \matc{\frac{3+\sqrt{33}}{6}\\-1}$. Similarly, one eigenvector corresponding to eigenvalue $\frac{5+\sqrt{33}}{2}$ is $\vec v_2=\matc{\frac{3-\sqrt{33}}{6}\\-1}$ is one eigenvector with eigenvalue $\lambda_2 = \frac{5+\sqrt{33}}{2}$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: -> `Thus, we conclude that XXX is an eigenvector with eigenvalue YYY`

File: problemsets-mat223/modules/module15.tex [hunk 12]
```diff
@@ -121,23 +117,23 @@
...
+	We conclude, one eigenvector of $A$ with eigenvalue $\lambda_1=-1$ is $\mat{1\\-1}$. More generally, vectors $\vec v = k \mat{1\\-1}$ where $k \neq 0$ are eigenvectors of A with eigenvalue $-1$. 
 	
 	Similarly,
 	\[
 	\Null(A-\lambda_2I) = \Null\left(\mat{-3&2\\3&-2}\right)=
-	\Span\Set*{\mat{1\\-1}},
+	\Span\Set*{\mat{2\\3}},
 	\]
-	and so the eigenvectors corresponding to the eigenvalue $\lambda_2=4$ are $\vec v_2 = k\mat{2\\3}$ where $k \neq 0$.
+	and so the eigenvectors with the eigenvalue $4$ are $\vec v_2 = k\mat{2\\3}$ where $k \neq 0$.
```
Comment (siefkenj) [inline problemsets-mat223/modules/module15.tex]: Remove `the`

**Pull request merged**